spec/namespaces/neural.yaml

```diff
@@ -0,0 +1,107 @@
+  version: 1.0.0
+paths:
+  /_plugins/_neural/stats:
+    get:
+      operationId: neural.stats.0
+      x-operation-group: neural.stats
+      x-version-added: '3.0'
+      description: Provides information about the current status of the neural-search plugin.
+      parameters:
+        - $ref: '#/components/parameters/neural.stats::path.stat'
+        - $ref: '#/components/parameters/neural.stats::query.flat_stat_paths'
+        - $ref: '#/components/parameters/neural.stats::query.include_metadata'
+      responses:
+        '200':
+          $ref: '#/components/responses/neural.stats@200'
+  /_plugins/_neural/stats/{stat}:
+    get:
+      operationId: neural.stats.1
+      x-operation-group: neural.stats
+      x-version-added: '3.0'
+      description: Provides information about the current status of the neural-search plugin.
+      parameters:
+        - $ref: '#/components/parameters/neural.stats::path.stat'
+        - $ref: '#/components/parameters/neural.stats::query.flat_stat_paths'
+        - $ref: '#/components/parameters/neural.stats::query.include_metadata'
+      responses:
+        '200':
+          $ref: '#/components/responses/neural.stats@200'
+  /_plugins/_neural/{node_id}/stats:
+    get:
+      operationId: neural.stats.2
+      x-operation-group: neural.stats
+      x-version-added: '3.0'
+      description: Provides information about the current status of the neural-search plugin.
+      parameters:
+        - $ref: '#/components/parameters/neural.stats::path.node_id'
+        - $ref: '#/components/parameters/neural.stats::query.flat_stat_paths'
+        - $ref: '#/components/parameters/neural.stats::query.include_metadata'
+      responses:
+        '200':
+          $ref: '#/components/responses/neural.stats@200'
+  /_plugins/_neural/{node_id}/stats/{stat}:
+    get:
+      operationId: neural.stats.3
+      x-operation-group: neural.stats
+      x-version-added: '3.0'
+      description: Provides information about the current status of the neural-search plugin.
+      parameters:
+        - $ref: '#/components/parameters/neural.stats::path.node_id'
+        - $ref: '#/components/parameters/neural.stats::path.stat'
+        - $ref: '#/components/parameters/neural.stats::query.flat_stat_paths'
+        - $ref: '#/components/parameters/neural.stats::query.include_metadata'
+      responses:
+        '200':
+          $ref: '#/components/responses/neural.stats@200'
+components:
+  responses:
+    neural.stats@200:
+      description: Successful response containing neural stats
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/neural._common.yaml#/components/schemas/NeuralStats'
+  parameters:
+    neural.stats::path.node_id:
+      name: node_id
+      in: path
+      description: Comma-separated list of node IDs or names to limit the returned information; leave empty to get information from all nodes.
+      schema:
+        type: string
+      required: true
```

Comment:
is node_id required param?

Reply:
It's required in paths where node_id is in the url, otherwise it's not specified as a parameter